src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
   <string name="saving_details">Saving details</string>
   <string name="screenshots">Screenshots</string>
   <string name="search">Search</string>
-  <string name="select_the_mirror">Select the mirror</string>
-  <string name="show_more">Show more</string>
-  <string name="show_older_releases">Show older releases</string>
+  <string name="select_the_mirror">Select a mirror</string>
+  <string name="show_more">More</string>
+  <string name="show_older_releases">Show older versions</string>
   <string name="skip">Skip</string>
   <string name="socks_proxy">SOCKS proxy</string>
-  <string name="sort_order">Sort order</string>
+  <string name="sort_order">Sorting order</string>
   <string name="source_code">Source code</string>
   <string name="suggested">Suggested</string>
   <string name="sync_repositories">Sync repositories</string>
-  <string name="sync_repositories_automatically">Sync repositories automatically</string>
-  <string name="syncing">Syncing</string>
-  <string name="syncing_format">Syncing %s</string>
+  <string name="sync_repositories_automatically">Automatically</string>
+  <string name="syncing">Syncing…</string>
+  <string name="syncing_format">Syncing %s…</string>
   <string name="tap_to_install_description">Tap to install.</string>
   <string name="theme">Theme</string>
   <string name="uninstall">Uninstall</string>
   <string name="unknown">Unknown</string>
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    application from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest upgrades to unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about new applications</string>
+  <string name="update_notifications_summary">Show a notification when new versions are available</string>
```

Comment:
"Notifications about new applications" is wrong here. About which "new applications"? My program doesn't notify about any new applications, it notifies about new updates. This is why "Update notifications" was used here.

Again, let's keep "updates" instead of "new versions".

Reply:
Changed to "newer"
"Update notifications" sounds like it is "notifications" that are being updated.

Reply:
There is a subtitle, which can explain it. But I agree, "Update notifications" can be not a good option. "Notify about updates" maybe? It will be in imperative mood, like "sync repositories automatically".

Reply:
"Announce installable applications" or something?

Reply:
"Announce new updates" is better. But "Announce" itself is not good, because user got used to "notification" word.